core/domain/stats_jobs_one_off.py

```diff
@@ -265,16 +270,7 @@ def reduce(exp_id, stringified_values):
                     state_stats_mapping[state_name] = (
                         stats_domain.StateStats.create_default())
             else:
-                try:
-                    exp_commit_log = (
-                        exp_models.ExplorationCommitLogEntryModel.get(
-                            'exploration-%s-%s' % (exp_id, version)))
-                except Exception:
-                    yield (
-                        "ERROR: Missing CommitLogEntryModel for exploration "
-                        "%s, version %s." % (exp_id, version))
-                    return
-                change_list = exp_commit_log.commit_cmds
+                change_list = snapshots_by_version[version-1]['commit_cmds']
```

Comment:
Add spaces around "-".

How do you know that the entry at index "version - 1" is the entry corresponding to "version - 1"? The return value from get_snapshots_metadata() is a list.

Reply:
Hmm. Expected that to be sorted by version number, and that seems to be the case since tests pass, but, lets sort it by version number anyway.

Reply:
Oh, actually, that was my error, sorry -- I didn't look far enough up the code to see the definition of version_numbers. It's fine!